src/llm/http_llm_calculator.cc

```diff
@@ -176,11 +176,17 @@ class OpenAIChatCompletionsRequest {
                 return absl::InvalidArgumentError("stream_options is not an object");
             auto streamOptionsObj = it->value.GetObject();
 
+            int streamOptionsFound = 0;
             it = streamOptionsObj.FindMember("include_usage");
             if (it != streamOptionsObj.MemberEnd()) {
                 if (!it->value.IsBool())
                     return absl::InvalidArgumentError("stream_options.include_usage is not a boolean");
                 this->streamOptions.includeUsage = it->value.GetBool();
+                streamOptionsFound++;
+            }
+
+            if (streamOptionsObj.MemberCount() > streamOptionsFound) {
+                return absl::InvalidArgumentError("Found unexpected stream options. Properties accepted in stream_options: include_usage");
             }
```

Comment:
```c++
it = streamOptionsObj.FindMember("include_usage");
if (it != streamOptionsObj.MemberEnd())
  return absl::InvalidArgumentError("Found unexpected stream options. Properties accepted in stream_options: include_usage");
if (!it->value.IsBool())
  return absl::InvalidArgumentError("stream_options.include_usage is not a boolean");
this->streamOptions.includeUsage = it->value.GetBool()
```

I think it is shorter and cleaner as we avoid multi-line indent

Reply:
Let's stick with this current approach as it's more extensible in case there are new options we want to add